.github/workflows/run-tests.yml

```diff
@@ -9,10 +9,12 @@ jobs:
       fail-fast: true
       matrix:
         os: [ubuntu-latest]
-        php: [7.4, 8.0]
-        laravel: [7.*, 8.*]
+        php: [7.4, 8.0, 8.0]
```

Comment:
I'm not sure why it duplicates the PHP version here, but you can remove it. We'll get this fixed up for next time. 👍

Reply:
Yeah. Sure. Thanks 👍